bundles/auth/org.eclipse.smarthome.auth.oauth2client/src/main/java/org/eclipse/smarthome/auth/oauth2client/internal/OAuthClientServiceImpl.java

```diff
@@ -26,6 +26,7 @@
 import org.eclipse.jdt.annotation.NonNull;
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.util.StringUtil;
```

Comment:
Are you sure you want to use such a util class from Jetty? Maybe rather go for Apache Commons Lang...

Reply:
I used it because that package is already used and the apache is not, so it would add a new library.